UI/web-src/ui/directives/liveEditor/dualPaneEditor.html

```diff
@@ -8,7 +8,7 @@
             </md-card-title>
             <md-card-content flex layout="column" layout-align="start center">
                 <ma-ace-editor flex style="height:450px" ng-model="text" ng-model-options="{debounce: 1000}"></ma-ace-editor>
-				<md-button class="md-raised ma-copy-button" ui-sref="ui.settings.editPages({markup: text})"><span ma-tr="ui.app.copyToNewPage"></span></md-button>
+				<md-button class="md-raised ma-copy-button" ui-sref="ui.setting.editPages({markup: text})"><span ma-tr="ui.app.copyToNewPage"></span></md-button>
```

Comment:
Is this a mistake? Most of the other `ui.settings` were changed to `ui.system`.

Reply:
Yes it was a mistake it should be "ui.uiSettings", i fix it.  We have now  "ui.uiSettings" and "ui.settings":
<img width="822" alt="Screenshot 2023-03-24 at 12 17 44 PM" src="https://user-images.githubusercontent.com/87542019/227608236-aa79cf49-cb45-4dda-81ea-38a0b990a801.png">
<img width="924" alt="Screenshot 2023-03-24 at 12 19 21 PM" src="https://user-images.githubusercontent.com/87542019/227608249-92e99e4c-a728-4907-a24f-572029ff37b1.png">
<img width="262" alt="Screenshot 2023-03-24 at 12 21 10 PM" src="https://user-images.githubusercontent.com/87542019/227608364-c62d1f36-6fa2-4096-8f5e-dee09c36c86d.png">

Reply:
There were multiple instances of this, did you fix them all up?

Reply:
Yes I fixed all of them.